test-harness/src/test/java/com/netflix/conductor/tests/utils/MockIndexDAO.java

```diff
@@ -18,8 +18,8 @@
  */
 package com.netflix.conductor.tests.utils;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
```

Comment:
One of the things that will be added to the Conductor code base in near future is `FindBugs` and `PMD`, which will help catch the smaller static analysis sooner. Until then please take out the import with *